### DISCUSSION TITLE
Guest mode not available anymore?

### DISCUSSION THREAD

**Discussion by caballerofelipe** [OpenBB-finance/OpenBB#5585] [Q&A] [answered]
I was using version 3.0.1 and I was able to use it as guest. Is this functionality dead? Or how do I enable guest mode?

**Comment by deeleeramone** [comment, marked as answer]
> I was using version 3.0.1 and I was able to use it as guest. Is this functionality dead? Or how do I enable guest mode?

Hello,

You are correct, guest mode is no longer available in the released packaged versions.  In order to use in Guest Mode, you will need to install via source or PyPI.

**Reply by caballerofelipe** [reply]
> Thanks. Where can I find the instructions on source and PyPI installing?

Never mind... found it:
- PyPI: https://docs.openbb.co/terminal/installation/pypi
- Source: https://docs.openbb.co/terminal/installation/source
